test/helpers/utils.js

```diff
@@ -332,7 +332,12 @@ async function mintToken(contract, contractOwner, recipient, spender, amount) {
     value: parseEther("0.5"),
   });
 
-  await contract.connect(tokenOwnerSigner).mint(recipient.address, amount);
+  if(contract.address == USDC_ADDRESS){
+    await contract.connect(tokenOwnerSigner).transfer(recipient.address, amount);
+  }else{
```

Comment:
Nit: spacing. Use [Prettier](https://prettier.io/) in your IDE for formatting JS code!